test/scaladoc/scalacheck/HtmlFactoryTest.scala

```diff
@@ -819,4 +819,13 @@ object Test extends Properties("HtmlFactory") {
     }
 
   }
+
+  property("SI-9599 Multiple @todo formatted with comma on separate line") = {
+    createTemplates("SI-9599.scala")("X.html") match {
+      case node: scala.xml.Node =>
+        node.toString.contains(
```

Comment:
Here, could you please check if `node.text.contains("todo1todo2todo3")` instead of hard-coding the html backend output?